File: areEquallyStrong/areEquallyStrong.py
def areEquallyStrong(a0, a1, b0, b1):
    return max(a0, a1) == max(b0, b1) and min(a0, a1) == min(b0, b1)


print(areEquallyStrong(10, 15, 15, 10))
print(areEquallyStrong(15, 10, 15, 10))
print(areEquallyStrong(15, 10, 15, 9))

File: Completed/js/areEquallyStrong/areEquallyStrong.js
function areEquallyStrong(a0, a1, b0, b1) {
    return Math.max(a0, a1) == Math.max(b0, b1) && Math.min(a0, a1) == Math.min(b0, b1);
}

console.log(areEquallyStrong(10, 15, 15, 10))
console.log(areEquallyStrong(15, 10, 15, 10))
console.log(areEquallyStrong(15, 10, 15, 9))